chapters/en/slides/module2/module2_29.Rmd

```diff
@@ -58,24 +65,18 @@ Notes: Script here
 
 ---
 
-Let's return to the question we asked at the beginning slides of exercise 25: 
-
-_*Which manufacturer has the highest mean sugar content?*_   
+Let's learn some other useful abilities of `altair`. When specifying `x` and `y` in the `encode(..)` function, altair allows us to specify what data type to expect. See the table below for a description of the data types available and how to specify them.
 
-A nice way of showing our results would be to graph this. A bar chart like this should do the trick! 
+| Data Type   | Shorthand Code | Description |
+| ----------- | ----------- | ----------- |
+| Quantitative | `Q` |  a continuous real-valued quantity |
+| Ordinal   | `O` | a discrete ordered quantity |
+| Nominal | `N` | a discrete unordered quantity |
+| Temporal| `T` | a time or date value |
 
+Ordinal values imply that there is some natural ordering to the values. For example, the ratings a movie receives is on an ordinal scale since a five star rating is better than a single star rating. In contrast, there is no such natural ordering for nominal values. An example of this would be strings.
 
-```{python echo=FALSE, fig.width = 6, fig.height = 4}
-sugar_plot = (df.groupby(by='mfr')
-                .mean()
-                .loc[:,'sugars']
-                .plot.bar(title='Mean sugar content among manufacturers')
-              )
-sugar_plot.set_ylabel('Sugar content (in grams)', fontsize=12)
-sugar_plot = sugar_plot.set_xlabel('Manufacturer', fontsize=12)
-sugar_plot
-```
-
+Altair is sometimes smart enough to tell which data type the input is. For example, `strings` are considered as `nominal`, anything `numeric` will be considered as `quantitative` and `time` values are considered as `temporal`. We will see examples of temporal data types in module 8. However, these defaults may not always be correct. In the next slide, we will see an example of specifying data types when plotting.
```

Comment:
we do not explain strings until module 4 so this needs to be amended. Give examples instead of just "strings" or "numeric" also "time" could be considered a string in some cases so avoid being extremely blank and white about this.